postgres/src/gen_test_data.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE CRYPTO_TRADES(
+    T_DATE date,
+    T_TS bigint,
+    T_ID varchar(20),
+    T_INSTRUMENT varchar(10),
+    T_EXCHANGE text,
+    T_PRICE double precision,
+    T_SIZE real
+);
+
+COPY CRYPTO_TRADES
+FROM '/tmp/psql_data/crypto_trades.csv'
```

Comment:
Could also have this command copy from the example URL.  https://stackoverflow.com/questions/41696675/how-to-copy-a-csv-file-from-a-url-to-postgresql